### ISSUE TITLE
asp.net core 2.1 UseHsts() naming conflict

### ISSUE THREAD

**Issue by dotnetshadow** [juunas11/aspnetcore-security-headers#20]
Hi there,

Great library, I noticed with asp.net core 2.1 that when I'm using your library that I need to fully qualify the name .UseHsts() method that asp.net core 2.1 uses

```csharp
HstsBuilderExtensions.UseHsts(app);
```

I can't use 
```csharp
app.UseHsts();
```

**Comment by juunas11** [comment]
Great minds think alike etc :smiley: I could rename to like `UseStrictTransportSecurity()`, but it'd be a breaking change :(

**Comment by aerichmond** [comment]
Yes.  Even though it's a breaking change I think it should be renamed.
